### ISSUE TITLE
Feature request: Add automatic backup of sqlite database

### ISSUE THREAD

**Issue by VibrantTreeStump** [brandonp2412/Flexify#83]
It would be nice if there were an option to enable automatic backups. Optionally maybe daily/weekly.
If you're open to that, I would suggest naming the backed up database `YYYY-MM-DD_flexify.sqlite`

**Comment by brandonp2412** [comment]
I actually tried this on and off for several months to no avail. I would back up the first time just fine, and on subsequent manual requests it would work, however doing it on a schedule would fail silently. @josephnglynn tried helping me out with a PR that persisted the permissions settings but I believe I already tried that a long while ago and since gave up.

**Comment by josephnglynn** [comment]
I thought the changes I made in this [branch](https://github.com/josephnglynn/Flexify/tree/fix_auto_backup) worked, but I never submitted a PR, because I wasn't sure how and when backups should be taken.

I tried updating the branch to test it again, but I currently don't have access to a computer for the next couple of weeks and trying to make changes on a phone is finicky at best.

Maybe someone can test that branch on an emulator and see if it works?
